build.android

```diff
@@ -42,7 +42,8 @@ cmake "$@" \
     -DANDROID_NDK=${NDK_ROOT}                                               \
     -DANDROID_ABI=${abi_name}                                               \
     -DANDROID_PLATFORM=android-24                                           \
-    -DOCOS_BUILD_ANDROID=ON                                                 \
     -DANDROID_NDK_VERSION=${CURRENT_NDK_VERSION}                            \
-    ../../.. && cmake --build . --config Release --parallel
+    -DOCOS_BUILD_ANDROID=ON                                                 \
+    -DOCOS_ENABLE_SELECTED_OPLIST=ON                                        \
```

Comment:
maybe we can keep this argument outside of the script so the script is more general.
instead, we can call `tools/android.package.sh -DOCOS_ENABLE_SELECTED_OPLIST=ON`. it should pass the arguments through to this cmake command.